### ISSUE TITLE
Question: !.20.1 Version?

### ISSUE THREAD

**Issue by penturt** [VazkiiMods/Morph-o-Tool#80]
Any plans to update to 1.20 and above? This is a great mod, and it sucks not having it available.

**Comment by choekstr** [comment]
It's an incredible mod and really tough playing 1.20 packs without it.  Hoping Vazkii can find some time to port this up to 1.20

**Comment by ratkinsonuk** [comment]
+1 for this please Vazkii

**Comment by KatoNamus** [comment]
+1, please update

**Comment by jacobwroper** [comment]
I'll add that I really miss morph-o-tool in 1.20; no expectations, but if it got an update, I wouldn't mind :)

**Comment by Uraneptus** [comment]
I'm on it ^^

**Comment by ratkinson-prh** [comment]
Guys, thanks for taking the time to port this over. Looking at the diffs, it clearly wasn't a 5 minute task.

On a side note, can I ask where you get/got the information from regarding change of function names, new classes, etc within the Minecraft framework?  Is this documented somewhere?

Cheers, Rob.

**Comment by ratkinson-prh** [comment]
It definitely was, thanks. I'd hoped MC/Forge had something more structured in place, but as I've never really seen anything on the web, I suspected developers were left to fend for themselves. I certainly don't envy you guys having to keep chasing updates when you're not given the right information to deal with them properly.

I often contemplate writing some MC code myself, but the learning curve to understand and implement the framework is just too much. I've spent a lifetime learning programming languages, and I don't think I've got it in me to start again with the MC API's.

Cheers, Rob.
